packages/salesforcedx-vscode-apex-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -808,14 +808,14 @@ export async function processBreakpointChangedForCheckpoints(
   for (const bp of breakpointsChangedEvent.removed) {
     if (bp.condition && bp.condition!.toLowerCase().indexOf(CHECKPOINT) >= 0) {
       await lock.acquire(CHECKPOINTS_LOCK_STRING, async () => {
-        const breakpointId = (bp as any)._id;
+        const breakpointId = bp.id;
```

Comment:
this was the fix for the breakpoints panel. I'm not sure when this changed in the vscode spec, but def calls to light not referencing private properties in 3rd party libs as a best practice.